spec/controllers/users_controller_spec.rb

```diff
@@ -44,6 +46,32 @@
         expect(assigns(:user)).to be_instance_of(User)
       end
     end
+    
+    context "sending emails" do
+      context "valid input details" do
+        
+        before { post :create, user: { email_address: 'knoxjeffrey@outlook.com', password: 'password', full_name: 'Jeff Knox' } }
+        after { ActionMailer::Base.deliveries.clear }
+        
+        it "sends out the email" do
+          expect(ActionMailer::Base.deliveries.last.to).to eq(['knoxjeffrey@outlook.com'])
+        end
+      
+        it "sends email containing the users full name" do
+          expect(ActionMailer::Base.deliveries.last.body).to include("Jeff Knox")
+        end
```

Comment:
That's better, I was trying to test the string and the url with ```include("Reset My Password", href: "http://localhost:3000#{password_resets_path}/#{valid_user.reload.token}")``` and in that case it wouldn't pass unless it had the http://localhost:3000 part in the url.